docs/src/manifolds/stiefel.md

```diff
@@ -0,0 +1,23 @@
+# Stiefel
+
+```@autodocs
+Modules = [Manifolds]
+Pages = ["Stiefel.jl"]
+Order = [:type, :function]
+```
+
+## Literature
+
+T. Kaneko, S. Fiori, T. Tanaka: “Empirical Arithmetic Averaging over the
```

Comment:
Yes, we can do that. I am currently using a different format in doc strings but I think that is necessary since they should be readable on REPL. I will reformat these